src/vsphere_cpi/lib/ruby_vim_sdk/soap/stub_adapter.rb

```diff
@@ -18,7 +18,6 @@ def initialize(api_uri, version, http_client)
       def invoke_method(managed_object, method_info, arguments, outer_stub = nil)
         outer_stub = self if outer_stub.nil?
         headers = {'SOAPAction' => @version_id,
-                   'Accept-Encoding' => 'gzip, deflate',
```

Comment:
Is the removal of this line necessary given that we now set `@backing_client.transparent_gzip_decompression = true`?

Reply:
It's not necessary to remove that line, but the library handles setting that header when you set `transparent_gzip_decompression` to `true`, so -in my opinion- it's best if we don't set that header.